charts/risingwave/values.yaml

```diff
@@ -1389,3 +1389,21 @@ cloud:
       ## This only works when endpoint is not set and region is set.
       ##
       useVPCEndpoint: false
+
+## @section license. Enterprise license key configuration for RisingWave enterprise.
+##
+
+license:
```

Comment:
Ditto

Reply:
This is the license block, not `license key` block. I think the `key` field below stands for `licenseKey`.